controllers/comments.js

```diff
@@ -0,0 +1,6 @@
+const {selectCommentByArticleId} = require("../models/comment-models")
+
+exports.getCommentByArticleId = (req,res,next) => {
+
+    selectCommentByArticleId();
```

Comment:
we still want to send the comments from here to the client in a .then() as that's the controller's job